common/visualization.js

```diff
@@ -29,18 +29,18 @@
  */
 window.OpenSeadragonizer = {
     open: function (url) {
-        var loaderDiv = document.getElementById("loader");
+        var popup = document.getElementById("popup");
+        popup.style.display = "none";
 
-        url = url || OpenSeadragon.getUrlParameter("img");
+        url = url || OpenSeadragon.getUrlParameter("img") ||
```

Comment:
This logic won't work so well in the error case, since the URL parameter will still be there, so the new URL they input will never be used. Perhaps better would be to put the `document.getElementById("url").value` into the onclick in the HTML.